src/PartielsPy/Partiels.py

```diff
@@ -0,0 +1,49 @@
+"""A main class for Partiels Wrapper"""
+
+import os
+import platform
+import shutil
+
+
+class Partiels:
+    def __init__(self):
+        self.__executable_path = self.__findExecutablePath()
+
+    def __findExecutablePath(self):
+        name = "Partiels"
+        system = platform.system()
+        path = os.environ.get("PARTIELS_PATH")
```

Comment:
I think you should use something like `shutil.which(name, os.environ.get("PARTIELS_PATH"))` because PARTIELS_PATH should be the path to the directory or the bundle of the application.